.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Bug report
+about: Create a report to help us reproduce and correct the bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+#### Describe the bug
+A clear and concise description of what the bug is.
+
+#### Steps/Code to Reproduce
+<!--
+Example:
```

Comment:
One thing we usually end up writing as a comment is that the code should be reproducible. I know the title says it, but people put a piece of code which we can't reproduce, and we can't do much about it

Reply:
it'd be nice to add a comment here explaining that

Reply:
This would be a good addition +1